src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -57,7 +54,9 @@ public void SaveSequenceLoadedInEditor()
             }
             
             sequenceEditor.SetActive(false);
-            LoadSequences();
+            
+            var sequences = ResolveSequenceFiles();
+            InstantiateSequences(sequences);
         }
```

Comment:
Changed this to the `Start` method so that I could have more control of initialization during tests. The script functions the same when used for real